index.ejs

```diff
@@ -1,127 +1,98 @@
-<% var item, key %><%
-htmlWebpackPlugin.options.appMountIds = htmlWebpackPlugin.options.appMountIds || [] %><%
-htmlWebpackPlugin.options.lang = htmlWebpackPlugin.options.lang || "en" %><%
-htmlWebpackPlugin.options.links = htmlWebpackPlugin.options.links || [] %><%
-htmlWebpackPlugin.options.meta = htmlWebpackPlugin.options.meta || [] %><%
-htmlWebpackPlugin.options.scripts = htmlWebpackPlugin.options.scripts || []
-%><!DOCTYPE html>
-<html lang="<%= htmlWebpackPlugin.options.lang %>"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>>
-  <head>
-    <meta charset="utf-8">
-    <meta content="ie=edge" http-equiv="x-ua-compatible"><%
-
-    if (htmlWebpackPlugin.options.baseHref) { %>
-    <base href="<%= htmlWebpackPlugin.options.baseHref %>"><%
-    } %><%
-
-    if (Array.isArray(htmlWebpackPlugin.options.meta)) { %><%
```

Comment:
I would like to keep the ability to add meta tags as an array -- a user can add as many of the unanticipated kind of meta tags as they want.  This package has drifted this direction as more and more people have wanted it to do more things:  Just allow general setting of html via json.